lib/receptor_controller/client/response_worker.rb

```diff
@@ -65,7 +65,6 @@ def stop
     # @param timeout_callback [Symbol] name of receiver's method processing timeout [optional]
     # @param error_callback [Symbol] name of receiver's method processing errors [optional]
     def register_message(msg_id, receiver, response_callback: :response_success, timeout_callback: :response_timeout, error_callback: :response_error)
-      logger.debug("Receptor response: registering message #{msg_id}")
```

Comment:
@lindgrenj6 This seems important for now since we are debugging it will tell us when the msg_id was registered.

Reply:
@lindgrenj6 I see you have moved the message up. I am good with this change